docs/analyzer/user_guide.md

```diff
@@ -245,6 +246,9 @@ analyzer arguments:
                         For more information see the docs: https://github.com/
                         Ericsson/codechecker/tree/master/docs/config_file.md
                         (default: None)
+  --cppcheck-args CPPCHECK_ARGS_CFG_FILE
+                        Configuration file to pass cppcheck command line
```

Comment:
```suggestion
                        Configuration file to pass to the Cppcheck command line
```

Reply:
oh this is a good catch, this paramater is non existent now.